lib/css_parser/rule_set.rb

```diff
@@ -647,14 +652,21 @@ def parse_declarations!(block) # :nodoc:
       return unless block
 
       continuation = nil
-      block.split(/[;$]+/m).each do |decs|
```

Comment:
this was splitting on newline or semicolon, is this still there ?
did you find good test coverage or should we add some more for edge-cases before this refactor, for example "\n\n;;;\n\n" and other nonsense to make sure it does not blow up